### PR TITLE
write valid files also with small screen width

### PR DESCRIPTION
**Pull request by ThomasBreuer**
The functions 'COHOMOLO.WritePermGroup', 'COHOMOLO.WriteFPGroup', 'COHOMOLO.WriteMatrices' repeatedly called 'AppendTo' for writing (new) files.
This caused the insertion of trailing backslashes if the screen width is 72 (or a little bit smaller or bigger, depending on the input), with the effect that the files cannot be processed by the standalone.

Now the code uses 'FileString', which does not depend on the screen width.
A test was added that had failed previously.

### CODE REVIEW

**Comment by markuspf** [comment]
The test failure now is due to GAP's new build system. I will fix this imminently.

**Comment by markuspf** [comment]
If you rebase your code on the current master, tests should be run on travis.

File: tst/testall.tst [hunk 1]
```diff
@@ -73,5 +73,16 @@ gap> Ex:= NonsplitExtension( chr, [1,2] );;
 gap> SE:= Subgroup( Ex, [Ex.4,Ex.5,Ex.6,Ex.7,Ex.8] );;
 gap> Index( Ex, SE );
 1080
+gap> Read( Filename( TestDir, "sz8" ) );
```
Comment (fingolfin) [inline tst/testall.tst]: That doesn't work, `TestDir` must be replaced by `testdatadir`.

**Comment by fingolfin** [comment]
Fixed via PR #10

**Pull request closed without merging**